src/dlt_packet_slice.rs

```diff
@@ -2,6 +2,108 @@ use crate::verbose::VerboseIter;
 
 use super::*;
 
+#[cfg(feature = "std")]
+const SET_LOG_LEVEL: &[u8] = "set_log_level".as_bytes();
+#[cfg(feature = "std")]
+const SET_TRACE_STATUS: &[u8] = "set_trace_status".as_bytes();
+#[cfg(feature = "std")]
+const GET_LOG_INFO: &[u8] = "get_log_info".as_bytes();
+#[cfg(feature = "std")]
+const GET_DEFAULT_LOG_LEVEL: &[u8] = "get_default_log_level".as_bytes();
+#[cfg(feature = "std")]
+const STORE_CONFIGURATION: &[u8] = "store_configuration".as_bytes();
+#[cfg(feature = "std")]
+const RESET_TO_FACTORY_DEFAULT: &[u8] = "reset_to_factory_default".as_bytes();
+#[cfg(feature = "std")]
+const SET_MESSAGE_FILTERING: &[u8] = "set_message_filtering".as_bytes();
+#[cfg(feature = "std")]
+const SET_DEFAULT_LOG_LEVEL: &[u8] = "set_default_log_level".as_bytes();
+#[cfg(feature = "std")]
+const SET_DEFAULT_TRACE_STATUS: &[u8] = "set_default_trace_status".as_bytes();
+#[cfg(feature = "std")]
+const GET_SOFTWARE_VERSION: &[u8] = "get_software_version".as_bytes();
+#[cfg(feature = "std")]
+const GET_DEFAULT_TRACE_STATUS: &[u8] = "get_default_trace_status".as_bytes();
+#[cfg(feature = "std")]
+const GET_LOG_CHANNEL_NAMES: &[u8] = "get_log_channel_names".as_bytes();
+#[cfg(feature = "std")]
+const GET_TRACE_STATUS: &[u8] = "get_trace_status".as_bytes();
+#[cfg(feature = "std")]
+const SET_LOG_CHANNEL_ASSIGNMENT: &[u8] = "set_log_channel_assignment".as_bytes();
+#[cfg(feature = "std")]
+const SET_LOG_CHANNEL_THRESHOLD: &[u8] = "set_log_channel_threshold".as_bytes();
+#[cfg(feature = "std")]
+const GET_LOG_CHANNEL_THRESHOLD: &[u8] = "get_log_channel_threshold".as_bytes();
+#[cfg(feature = "std")]
+const BUFFER_OVERFLOW_NOTIFICATION: &[u8] = "buffer_overflow_notification".as_bytes();
+#[cfg(feature = "std")]
+const SYNC_TIME_STAMP: &[u8] = "sync_time_stamp".as_bytes();
+#[cfg(feature = "std")]
+const CALL_SWC_INJECTIONS: &[u8] = "call_swc_injections".as_bytes();
+#[cfg(feature = "std")]
+const DEPRECATED_COMMAND_NAME: &[u8] = "deprecated_command_name".as_bytes();
+
+const OK: &[u8] = "ok".as_bytes();
+const NOT_SUPPORTED: &[u8] = "not_supported".as_bytes();
+const ERROR: &[u8] = "error".as_bytes();
+
+#[derive(Debug, Clone, Eq, PartialEq, Hash)]
+#[cfg(feature = "std")]
+pub struct ControlMessage<'a> {
```

Comment:
We either remove it for the start or change it be an enum with the known messages, e.g.:

```Rust
pub enum ControlMsgSlice<'a> {
    GetLogInfo(GetLogInfoSlice<'a>),
    ...
    Unknown{ service_id: u32, payload: &[u8] },
```

Reply:
I left this task open for a different PR.